assume_role.sh

```diff
@@ -1,14 +1,14 @@
-if [ "$#" -ne 2 ]
+if [ "$#" -ne 1 ]
 then
-  echo "Usage: source assume_role.sh [account_id] [role]"
+  echo "Usage: source assume_role.sh [role_arn]"
   exit 1
 fi
 
-ACCOUNT="$1"
-ROLE="$2"
+ROLE_ARN="$1"
 
-role_session_name=`cat /proc/sys/kernel/random/uuid 2>/dev/null || date | cksum | cut -d " " -f 1`
-aws_creds=$(aws sts assume-role --role-arn arn:aws:iam::${ACCOUNT}:role/$ROLE --role-session-name $role_session_name --duration-seconds 3600 --output json)
+role_session_name="AWSCLIResourceSession"
+# 15 minutes should be sufficient to execute most AWS commands
```

Comment:
While your comment is correct, I'd rather not address multiple issues in the same commit/PR and keep the duration the same.